README.md

```diff
@@ -19,8 +19,13 @@ A collection of **100% FREE** to use portfolio website templates.
 ## Templates
 
 - [Hacker](https://ossphilippines.github.io/freefolio/hacker)
+![image](https://user-images.githubusercontent.com/58241136/191987391-e143a63e-3fba-494b-8061-33152cce33c4.png)
```

Comment:
Kindly just put the images in the root directory of the project under `images` folder and invoke them relatively like so:

```html
<image src="./image.png"/>
```

Compress the images first using https://tinypng.com/ and name each similar to the theme name, e.g. `hacker.png`